java/kotlin-extractor2/src/main/kotlin/entities/Class.kt

```diff
@@ -214,8 +214,8 @@ fun KotlinUsesExtractor.useClassSource(c: KaClassSymbol): Label<out DbClassorint
  */
 private fun tryReplaceType(
     cBeforeReplacement: KaClassSymbol,
-    argsIncludingOuterClassesBeforeReplacement: List<Nothing>?
-): Pair<KaClassSymbol, List<Nothing>?> {
+    argsIncludingOuterClassesBeforeReplacement: List<List<KaTypeProjection>>?
```

Comment:
Note this `List<List<KaTypeProjection>>` format occurs quite a lot, and represents the type-args of outer classes, if any.